mycelo/genesis/genesis.go

```diff
@@ -20,15 +20,16 @@ var genesisMsgHash = common.HexToHash("ecc833a7747eaa8327335e8e0c6b6d8aa3a38d006
 
 // CreateCommonGenesisConfig generates a config starting point which templates can then customize further
 func CreateCommonGenesisConfig(chainID *big.Int, adminAccountAddress common.Address, istanbulConfig params.IstanbulConfig) *Config {
-	genesisConfig := BaseConfig()
+	gingerbreadBlock := newBigInt(2)
+	genesisConfig := BaseConfig(gingerbreadBlock)
```

Comment:
Is this something we really want, or is it just included temporarily for testing the change?

Reply:
Temporary for testing